src/basic/config.cpp

```diff
@@ -143,6 +143,10 @@ Config::Config(int argc, const char **argv, bool check_io)
 	makedb.add()
 		("in", 0, "input reference file in FASTA format", input_ref_file);
 
+	Options_group cluster("Cluster options");
+	cluster.add()
+		("calgo", 0, "Clustering algorithm (0=multi-step/1=transitive-closure)", cluster_algo, -1);
```

Comment:
A string option would be prefered (more intuitive for the user).

Reply:
I agree. I thought about three possibilities, let me know what you think/want.
1) a specific keyword for each clustering algorithm available, e.g. --MCL or --Superfast-fuzzy-clustering-yeah
2) repurposing the --algo option which takes a keyword argument "MCL" or "Superfast-fuzzy-clustering-yeah"
3) Similar to the above with an extra option (not --calgo but) --clustering-algo with the aforementioned keyword argument

I personally would tend to option 3 (also because I am not sure you may want to use both --algo and --clustering-algo in the same run). If you have a different opinion/idea, please let me know.

Reply:
I would prefer option 3 as well.